src/o365/spfx/commands/project/project-rename.ts

```diff
@@ -90,81 +90,101 @@ class SpfxProjectRenameCommand extends BaseProjectCommand {
       }
     }
 
-    const filePaths: string[] = [
-      path.join(this.projectRootPath, 'package.json'),
-      path.join(this.projectRootPath, '.yo-rc.json'),
-      path.join(this.projectRootPath, 'config/package-solution.json'),
-      path.join(this.projectRootPath, 'config/deploy-azure-storage.json'),
-      path.join(this.projectRootPath, 'README.md')
-    ];
+    const replacePackageJsonContent = (filePath: string) => {
+      if (!fs.existsSync(filePath)) {
+        return;
+      }
+      const existingContent = fs.readFileSync(filePath, 'utf-8');
```

Comment:
To avoid the CLI crash on exceptions, let's wrap reading file contents, deserializing JSON and writing the file in a `try..catch` block and break the command execution on an exception

Reply:
@waldekmastykarz Just to be clear, the command execution should break with an error message if there is an exception in updating any of these files?

Reply:
Yes, so rather than returning an unhandled exception, we should call `cb()` (the action callback) passing the error message from the exception wrapped in `CommandError` as its argument.